content/en/docs/v3.6/op-guide/kubernetes.md

```diff
@@ -0,0 +1,438 @@
+---
+title: Run etcd clusters as a Kubernetes StatefulSet
+weight: 4200
+description: Running etcd as a Kubernetes StatefulSet
+---
+
+Below demonstrates how to perform the [static bootstrap process](../clustering/#static) as a Kubernetes StatefulSet.
+
+# Example Manifest
+This manifest contains a service and statefulset for deploying a static etcd cluster in kubernetes.
+
+If you copy the contents of the manifest into a file named `etcd.yaml`, it can be applied to a cluster with this command.
+```
+# kubectl apply -f etcd.yaml
```

Comment:
Nitpick - for documentation I prefer long param names that can be understood at a glance without looking them up. Obviously this example is a trivial case but it's a general rule I try to follow for users less familiar with command line tools.  Can you please apply to this and other commands in this doc.

```suggestion
# kubectl apply --filename etcd.yaml
```